src/sentry/static/sentry/app/options.jsx

```diff
@@ -102,8 +102,9 @@ const definitions = [
     key: 'beacon.anonymous',
     label: 'Usage Statistics',
     component: RadioBooleanField,
-    yesLabel: 'Send my contact information along with usage statistics',
-    noLabel: 'Please keep my usage information anonymous',
+    noLabel: 'Send my contact information along with usage statistics',
```

Comment:
i get why this is swapped, but it's super confusing on first encounter.

Reply:
you mean that no is positive and yes is negative? I agree but dont have a great answer

Reply:
A comment here would help